src/v3/src/transformer/selectAuthenticator/utils.ts

```diff
@@ -238,33 +273,35 @@ const formatAuthenticatorOptions = (
       return !isDup;
     })
     .map((option: IdxOption, index: number) => {
-      const authenticatorKey = option.relatesTo?.key as string;
+      const authenticator = option.relatesTo;
+      const authenticatorKey = authenticator?.key as string;
       const id = getOptionValue(option.value as Input[], 'id')?.value;
       const methodType = getOptionValue(option.value as Input[], 'methodType')?.value;
       const enrollmentId = getOptionValue(option.value as Input[], 'enrollmentId')?.value;
+      const isAdditionalEnroll = isEnroll && authenticator
+        && isAuthenticatorAlreadyEnrolled(authenticator, authenticatorEnrollments);
       const AUTHENTICATORS_WITH_METHOD_TYPE = [
         AUTHENTICATOR_KEY.ON_PREM,
         AUTHENTICATOR_KEY.OV,
         AUTHENTICATOR_KEY.RSA,
       ];
       const AUTHENTICATORS_WITH_NO_TRANSLATE_CLASS = [
         AUTHENTICATOR_KEY.PHONE,
-        AUTHENTICATOR_KEY.CUSTOM_APP,
+        AUTHENTICATOR_KEY.EMAIL,
```

Comment:
Was `AUTHENTICATOR_KEY.EMAIL` just originally missing from this list or did you make a change that caused this?

Reply:
`AUTHENTICATOR_KEY.EMAIL` should be in the list after my changes in https://github.com/okta/okta-signin-widget/pull/3392
Otherwise english leaks tests for gen 3 will fail on mock `authenticator-verification-select-authenticator-with-email`
(For now english leaks tests  are not enabled for gen 3 in Bacon so I've tested manually)